CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/).
 
 ## [Unreleased]
 
+## [v1.1.0] - 2021-07-28
```

Comment:
Please add a diff link to v1.1.0 as `v1.0.1`.

Reply:
Got it.